pt_os_web_portal/miniscreen_onboarding_assistant/page_manager.py

```diff
@@ -23,13 +24,15 @@ def __init__(self, miniscreen, default_page_interval=1):
             self.set_current_page_to_next_page
         )
 
-        self.current_page_index = 0
+        self.current_page_index = int(
+            state.get("miniscreen_onboarding", "state", fallback=0)
```

Comment:
i don't think we should do this based on the index because we don't know if the restarted service will have the same schema. if we use page names then we can just keep them the same so extra pages don't cause problems later...